system/Debug/Exceptions.php

```diff
@@ -328,6 +336,29 @@ protected function determineCodes(Throwable $exception): array
         return [$statusCode, $exitStatus];
     }
 
+    private function isDeprecationError(int $error): bool
+    {
+        $deprecations = E_DEPRECATED | E_USER_DEPRECATED;
+
+        return ($error & $deprecations) !== 0;
+    }
+
```

Comment:
```suggestion
    /**
     * @return true
     */
```

Reply:
Rector removes the docblock as useless.

Reply:
Huh that's odd. We have `true` as a native type coming up soon so it clearly has value over just `bool`. @samsonasik any ideas?

Reply:
@MGatner That's possibly a bug on `RemoveUselessReturnTagRector`.

Could you create failing test case PR to https://github.com/rectorphp/rector-src/tree/4cecebfd7e86cafd9e09f8e1fd493046aa1181ea/rules-tests/DeadCode/Rector/ClassMethod/RemoveUselessReturnTagRector/Fixture ? You can copy one of fixture with `skip_` prefix.

That's should can be skipped via `@noRector` tag:

```php
    /**
     * @noRector \Rector\DeadCode\Rector\ClassMethod\RemoveUselessReturnTagRector
     * @return true
     */
```

or register `RemoveUselessReturnTagRector` on rector.php into skip config:

```php
$rectorConfig->skip([
     // ...
     RemoveUselessReturnTagRector::class => [
           __DIR__ . '/system/Debug/Exceptions.php'
    ],
]);
```

Reply:
Thanks @samsonasik. Today I learned about the `@noRector` tag: I've always wanted that!!

I probably won't get to this today but I can follow up with Rector at some point. For now let's ignore this and keep the docblock.